lib/ui/painting.dart

```diff
@@ -3376,9 +3401,19 @@ enum TileMode {
   /// 4.0 to 3.0, and so forth (and for linear gradients, similarly from in the
   /// negative direction).
   ///
+  /// An image filter will treat the image as tiled in an alternating forwards and
```

Comment:
nit: "An image filter will treat the image ..." makes me think `TileMode` is a property of the image, not the filter.

Reply:
"An image filter will treat its source as ..."?

Reply:
Updates pushed...